ariadne/transformations.py

```diff
@@ -159,14 +133,15 @@ class BaseFilter(BaseTransformer):
          track_col (string, 'track' by default): column with station identifiers
     """
 
-    def __init__(self, filter_rule, num_stations=None, keep_fakes=True, station_col='station', track_col='track',
+    def __init__(self, filter_rule, num_stations=None, keep_filtered=True, station_col='station', track_col='track',
                  event_col='event'):
         self.num_stations = num_stations
         self._broken_tracks = None
         self._num_broken_tracks = None
-        self.keep_fakes = keep_fakes
+        self.keep_fakes = True
         self.filter_rule = filter_rule
```

Comment:
where is this `filter_rule`?!

Reply:
why we need any filter rules in scalers?

Reply:
please check it one more time, i don't have it